src/crypto.rs

```diff
@@ -1,7 +1,165 @@
+use thiserror::Error;
 use ssz_rs::prelude::*;
+use rand::prelude::*;
+
+use blst::{min_pk as blst_core, BLST_ERROR};
+use crate::crypto::Error::SizeMismatch;
+use blst::min_pk::AggregateSignature;
```

Comment:
and the other one is that this line is used -- below you reach it via `blst_core` name so it is unused here.

if you fix both of these things the CI should pass

Reply:
fixed